tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>For before zipping, I think it's sufficient to just say that zipping is required.</p><p> </p><p>Also it's good to move the zipping code into a utility, to make this more readable. See the <a href="https://github.com/tensorflow/model-optimization/pull/388" target="_blank">pruning example</a> as a reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>